strato_query/api_query.py

```diff
@@ -162,6 +165,8 @@ def query_api_multiple(queries: Dict[str, APIQueryParams],
                 df_ = pandas.DataFrame(v)
                 df_.columns = [c.upper() for c in df_.columns]
                 df_dict[k] = df_
+            if time_between_chunks is not None and idx_chunk + chunksize < len(keys_list):
```

Comment:
what is order of operations here? Is the plus sign evaluated before the < ? Can you just put the expression in parens?

Reply:
Also I don't get what that line does anyway

Reply:
Yes.

Reply:
If the user has specified a time to sleep, and the current chunk submitted does not represent the final chunk, then the method will sleep for the specified time.